src/views/CoCurricularTranscript/Components/Activity/index.tsx

```diff
@@ -12,14 +12,28 @@ type Props = {
 
 const Activity = ({ description, sessions }: Props) => {
   const leaderSessions = sessions.filter((s) => s.Participation === Participation.Leader);
+  const memberSessions = sessions.filter((s) => s.Participation === Participation.Member);
+  const advisorSessions = sessions.filter((s) => s.Participation === Participation.Advisor);
+
   return (
     <div className={styles.experience_transcript_activities}>
       <div className={styles.organization_role}>{description}</div>
-      <div className={styles.date}> {formatDuration(sessions)} </div>
+      <div className={styles.date}> {formatFinalDuration(sessions)} </div>
+      {advisorSessions.length > 0 && (
+        <div className={styles.participation_line}>
+          <div className={styles.organization_role}>
+            Advisor ({formatDuration(advisorSessions)})
+          </div>
+        </div>
+      )}
       {leaderSessions.length > 0 && (
-        <div className={styles.leadership_line}>
-          <div className={styles.organization_role}>Leader</div>
-          <div className={styles.date}>{formatDuration(leaderSessions)}</div>
+        <div className={styles.participation_line}>
+          <div className={styles.organization_role}>Leader ({formatDuration(leaderSessions)})</div>
+        </div>
+      )}
+      {memberSessions.length > 0 && (
+        <div className={styles.participation_line}>
+          <div className={styles.organization_role}>Member ({formatDuration(memberSessions)})</div>
```

Comment:
For consistency and readability, it's probably best to format these the same as `Advisor` above.
```suggestion
          <div className={styles.organization_role}>
            Leader ({formatDuration(leaderSessions)})
          </div>
        </div>
      )}
      {memberSessions.length > 0 && (
        <div className={styles.participation_line}>
          <div className={styles.organization_role}>
            Member ({formatDuration(memberSessions)})
          </div>
```